api/core/helper/__init__.py

```diff
@@ -0,0 +1,3 @@
+from core.helper.flask_context import flask_context_manager
```

Comment:
I think we should avoid creating broad packages named `helper` or `utils`, as they tend to become catch-all directories that grow unwieldy over time.

Instead, I'd recommend placing `flask_context_manager` in a more specific location like `libs/flask_utils.py`. This approach offers several benefits:

1. **Better semantic grouping** - Flask-related utilities are clearly organized together
2. **Cleaner imports** - Direct imports are more explicit and easier to track
3. **Improved maintainability** - Avoids the "junk drawer" problem common with generic helper packages

The import would then be:

```python
from libs.flask_utils import flask_context_manager
```

This follows the principle of organizing code by domain/functionality rather than by generic categories.

Reply:
SGTM